CMakeLists.txt

```diff
@@ -1611,7 +1611,7 @@ endif()
 # HPX_WITH_ACTION_BASE_COMPATIBILITY: introduced in V1.4.0
 hpx_option(
   HPX_WITH_ACTION_BASE_COMPATIBILITY BOOL
-  "Enable deprecated action bases (default: ON)" ON ADVANCED
+  "Enable deprecated action bases (default: OFF)" OFF ADVANCED
```

Comment:
Note to self. This is tentative.

Reply:
This is fine, I think.

Reply:
This one is perhaps less contentious, but my comment referred to all of the options below as well (which clearly do seem contentious!).

Reply:
Closing this as resolved. We can discuss on #5084 if needed.